test/ensure_minimum_node_version.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-env node */
+/* eslint no-console:off */
```

Comment:
We should probably move these into test/.eslintrc, or in the ../.eslintrc.js w/ [overrides](https://eslint.org/docs/user-guide/configuring#configuration-based-on-glob-patterns).